src/ArborX_LinearBVH.hpp

```diff
@@ -232,15 +236,20 @@ BoundingVolumeHierarchy<MemorySpace, Enable>::BoundingVolumeHierarchy(
       "ArborX::BVH::BVH::calculate_scene_bounding_box");
 
   // determine the bounding box of the scene
+  Box bbox{};
   Details::TreeConstruction::calculateBoundingBoxOfTheScene(space, primitives,
-                                                            _bounds);
+                                                            bbox);
 
   Kokkos::Profiling::popRegion();
 
   if (size() == 1)
   {
     Details::TreeConstruction::initializeSingleLeafNode(
         space, primitives, _internal_and_leaf_nodes);
+    Kokkos::deep_copy(space,
+                      Kokkos::View<BoundingVolume, Kokkos::HostSpace>(&_bounds),
+                      Kokkos::View<BoundingVolume, MemorySpace>(
+                          &getBoundingVolume(getRoot())));
```

Comment:
Still don't quite get why this is needed now, when it wasn't before.

Reply:
Because we were getting it when we computed the bounding box of the scene but now we have to go fetch it on the device